README.md

```diff
@@ -125,13 +125,13 @@ Depending on the [.NET configuration provider](https://docs.microsoft.com/dotnet
     }
   },
   "AllowedHosts": "*",
-  "RepositoryOptions": {
-    "CosmosConnectionString": "<Your-CosmosDB-ConnectionString>",
-    "DatabaseId": "<Your-CosmosDB-DatabaseName>",
-    "ContainerId": "<Your-CosmosDB-ContainerName>",
-    "OptimizeBandwidth": true,
-    "ContainerPerItemType": true,
-    "AllowBulkExecution": true
+  "Values": {
```

Comment:
I see what you're saying here, however; this would only be applicable to Azure Functions. You could add another section here, that differentiates between service / web tier and functions - then provide example _appsettings.json_ files. That would make it more explicit.